llvm/lib/CodeGen/AsmPrinter/AccelTable.cpp

```diff
@@ -559,37 +587,61 @@ void llvm::emitDWARF5AccelTable(
     default:
       continue;
     }
-    CUIndex[CU.index()] = Count++;
+    CUIndex[CU.index()] = CUCount++;
     assert(CU.index() == CU.value()->getUniqueID());
     const DwarfCompileUnit *MainCU =
         DD.useSplitDwarf() ? CU.value()->getSkeleton() : CU.value().get();
     CompUnits.push_back(MainCU->getLabelBegin());
   }
 
+  for (const auto &TU : enumerate(TUs)) {
+    switch (TU.value()->getCUNode()->getNameTableKind()) {
+    case DICompileUnit::DebugNameTableKind::Default:
+      break;
+    default:
+      continue;
+    }
```

Comment:
Seems a bit convoluted (unless there's an expectation to add more here) compared to:
```
if (TU.value()->getCUNode()->getNameTableKind() != DICompileUnit::DebugNameTableKind::Default)
  continue;
```